projects:ad4696_iio, ad7134_iio: Added github docs

jansunil opened the pull request:
Added github docs for AD4696 and AD7134

D-Disha commented on doc/sphinx/source/projects/ad4696_iio/ad4696_iio.rst:
```diff
@@ -0,0 +1,98 @@
+
+* `AD4695 <https://www.analog.com/en/products/ad4695.html>`_ 
+* `AD4696 <https://www.analog.com/en/products/ad4696.html>`_ 
+
```
Can we also add the EVAL board links?

jansunil replied:
Yes, added them now

D-Disha commented on doc/sphinx/source/projects/ad4696_iio/ad4696_iio.rst:
```diff
@@ -0,0 +1,98 @@
+
+* Make below jumper settings on the board. Refer `EVAL-AD4696-FMCZ User Manual <https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad4696.html#eb-overview>`_  for more details.
+
+   .. image:: /source/projects/ad4696_iio/ad4696_jumper_connection.png
```
I suppose table should be added as done in ad4130 project

jansunil replied:
Added a table format now

SaikiranGudla commented on doc/sphinx/source/projects/ad4696_iio/ad4696_iio.rst:
```diff
@@ -0,0 +1,98 @@
+
+   .. image:: /source/projects/ad4696_iio/ad4696_jumper_connection.png
+      :width: 600
+
```
nit: extra line can be removed

jansunil commented:
changelog:
1) Added a table for jumper connections in AD4696
2) Added links to the EVAL boards supported, from the product page
3) Resolved merge conflicts in the index.rst file

D-Disha commented on doc/sphinx/source/projects/ad7134_iio/ad7134_iio.rst:
```diff
@@ -0,0 +1,94 @@
+
+* `AD7134 <https://www.analog.com/en/products/ad7134.html>`_ 
+* `AD4134 <https://www.analog.com/en/products/ad4134.html>`_ 
+
```
Can Eval board links be added here as well.

jansunil replied:
Relevant boards that can be used with this FW aren't released yet.

D-Disha replied:
Can we add a TODO then, like ad719x?

jansunil replied:
A JIRA has been added to track this - https://jira.analog.com/browse/PCSW-557

jansunil commented:
Hi All- any review comments on this PR?
